src/less/time-column.less

```diff
@@ -83,7 +83,7 @@
     left: 0;
     background-color: white;
     border-right: 1px solid @cell-border;
-    z-index: 10;
+    z-index: 120;
```

Comment:
these new z-index feel a bit arbitrary to me, it seems like calendars with many overlapping events or very wide calendars would hit this limit pretty easily? Or maybe i'm remembering how offset works wrong :P